recipe/meta.yaml

```
@@ -49,7 +49,7 @@ outputs:
         - libnvjitlink
```

We need to pin since `libnvjitlink`'s minor version is not constrained by `cuda_version`.
```suggestion
        - libnvjitlink =={{ cuda_version }}
```